easybuild/easyconfigs/c/Cantera/Cantera-2.3.0-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,57 @@
+easyblock = 'SCons'
+
+name = 'Cantera'
+version = '2.3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/Cantera/cantera'
+description = """Chemical kinetics, thermodynamics, and transport tool suite"""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'strict': True}
+
+source_urls = ['https://github.com/Cantera/cantera/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'Cantera-%(version)s_fix-bug-416.patch',
+    'Cantera-%(version)s_fix-test-reprod.patch',
+    'Cantera-%(version)s_fix-recent-Cython.patch',
+]
+checksums = [
+    '06624f0f06bdd2acc9c0dba13443d945323ba40f68a9d422d95247c02e539b57',  # v2.3.0.tar.gz
+    'cbf52d6a2e24967da42edd060ae4e4958bb052967790c061680bfc157edec6c2',  # Cantera-2.3.0_fix-bug-416.patch
+    '73b49e8fc73adf221b956980e5ef2a7808e6a81680f6e97ce846a159ae6d6966',  # Cantera-2.3.0_fix-test-reprod.patch
+    'e638ae44a25de9e3c38c21daa5d388f42106f9289a87f1fd01be3e985d7e7d54',  # Cantera-2.3.0_fix-recent-Cython.patch
+]
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('Boost', '1.65.1'),
+    ('SUNDIALS', '2.7.0'),
+    ('fmt', '3.0.2'),
+    ('googletest', '1.8.0'),
```

Comment:
build dep?

Reply:
Yeah, you're right, `googletest` only provides header files and static libraries, same for `fmt`.